### DISCUSSION TITLE
【提问】如何在github上的一个远程仓库中下载一个文件夹的文件

### DISCUSSION THREAD

**Discussion by xgdyp** [X-lab2017/open-wonderland#39] [2️⃣ 聊项目]
现在我找到了一种比较好用的方法，最后说～

**Comment by xgdyp** [comment]
目前网上看到的方法有使用svn下载的，不过我觉得有点麻烦。
通过codespace可以简化这种操作
假设我们想要下载PDF文件夹中的文件：
![image](https://user-images.githubusercontent.com/37795442/169485855-f65cfe57-fed5-4885-98df-7a8cab17cdca.png)
首先在repo页面按英文状态下`.`进入web editor
在web editor中可以直接下载该文件夹中的内容
![image](https://user-images.githubusercontent.com/37795442/169486229-5fcc2ffa-58ef-4496-901f-eba79be8042a.png)
这个操作在需要某个资源的时候很管用！

**Reply by will-ww** [reply]
挺不错！

不过，我们目前的方向应该是，把所有的pdf这类文件，全部放到语雀中，真个项目要代码化吧~

**Reply by xgdyp** [reply]
嗯嗯 举个例子～

**Reply by will-ww** [reply]
理解，好，棒！
